code/renderers/vue3/src/docs/sourceDecorator.test.ts

```diff
@@ -241,7 +239,7 @@ describe('Vue3: sourceDecorator->generateAttributesSource()', () => {
       generateAttributesSource(
         mapAttributesAndDirectives({ camelCaseBooleanArg: true }),
         { camelCaseBooleanArg: true },
-        [{ camelCaseBooleanArg: { type: 'boolean' } }] as ArgsType<Args>
+        [{ camelCaseBooleanArg: { type: 'boolean' } }] as any
```

Comment:
@kasperpeulen `ArgsType` is no longer exported from `jest-mock`. I couldn't find a replacement.

It looks unused, this argument, btw:
https://github.com/storybookjs/storybook/blob/6f0c2fc2912084577996348633e7242c7f93d12f/code/renderers/vue3/src/docs/sourceDecorator.ts#L61